static/app/views/onboarding/targetedOnboarding/welcome.tsx

```diff
@@ -21,16 +21,6 @@ import FallingError from 'sentry/views/onboarding/components/fallingError';
 
 import WelcomeBackground from './components/welcomeBackground';
 
-const easterEggText = [
```

Comment:
@Neo-Zhixing no more easter egg?

Reply:
It's still there, except now when you click the falling man it just falls..... silently.... and never to show up again.....

Reply:
big RIP